### DISCUSSION TITLE
Doc comment on Wrap-up (/developers/academy/zero_to_mvp/queries_2/wrap_up)

### DISCUSSION THREAD

**Discussion by giscus** [weaviate/weaviate-io#939] [Docs/blog comments]
# Doc comment on Wrap-up (/developers/academy/zero_to_mvp/queries_2/wrap_up)

&nbsp;&nbsp;Unit review

https://weaviate.io/developers/academy/zero_to_mvp/queries_2/wrap_up

**Comment by dkhundley** [comment]
Really good tutorial in general! I provided a brief bit of feedback on the respective pages to make this a little better for the learner, but generally speaking, I was really happy with this whole tutorial. Thank you all for putting it together! 🥳

**Comment by Taekyo-Lee** [comment]
I went through this course today. It really got me to understand the core concept of Weaviate. Worth your time.

**Comment by jcherreram** [comment]
Amazing tutorial and Database!

**Comment by legaltextai** [comment]
ditto. amazing tutorial. have gone through it non-stop. very well done.
